packages/web-components/src/utils/utils.ts

```diff
@@ -18,19 +18,21 @@ export function getSlottedNodes(
   // A `<style>` element and a `<slot>` element
   // Chrome, Firefox, Safari, Edge - literally every modern browser will use shadowDOM.
   // This is purely for IE compatibility
-  const hasShadowDOM =
-    Array.from(root.shadowRoot.childNodes).filter(
-      (node: any) => node.tagName === 'SLOT',
-    ).length > 0;
+  if (root.shadowRoot) {
```

Comment:
Why is this being added? It looks like this wouldn't return anything if `root.shadowRoot` is falsey, which I'm not sure is what we want.

Reply:
Katy wanted a test for the usage of this function in the light dom vs shadow dom (the last test focus on this). Right now if we don't include that wrapper for the function when being used with the light dom it will throw an error on `root.shadowRoot.childNodes` because the shadow dom doesn't exist. So therefore I added this to protect against someone using it for the light dom and because the use case of this function should only be for the shadow dom

Reply:
Light dom and shadow dom aren't mutually exclusive - a single component can make use of both. If we've got a web component that's using the shadow dom and it has a `<slot>` inside, then it is expecting to receive something from the light dom - a piece of HTML that is external to the component (and also not part of the component's shadow dom).

The `hasShadowDom` check is there so that `getSlottedNodes` will still return an array even in browsers that don't support shadow DOM functionality, like IE11. This `if (root.shadowRoot)` change will break components that rely on `getSlottedNodes` when viewed in IE11. For example, when you click a `<va-accordion-item>` it won't expand to reveal its slotted content.

Reply:
Sounds good let me remove and adjust the tests